src/main/java/com/spotify/github/v3/checks/CheckSuite.java

```diff
@@ -36,4 +37,7 @@ public interface CheckSuite {
    * @return the integer
    */
   Integer id();
+
+  @Nullable
```

Comment:
We use `Optional<T>` instead of `@Nullable` for optional values in all other events. Could you change this to be consistent with the rest of the codebase?

Reply:
thanks! will be more careful next time.